cli/main.rs

```diff
@@ -217,7 +218,10 @@ pub fn create_main_worker(
     }
   } else if let Some(config_file) = &ps.maybe_config_file {
     // otherwise we will use the path to the config file
-    config_file.path.to_str().map(|s| s.to_string())
+    specifier_to_file_path(&config_file.specifier)
+      .ok()
+      .map(|p| p.to_str().map(|s| s.to_string()))
+      .flatten()
```

Comment:
It seems like it would be better to just do `config_file.specifier.to_string()` here? I didn't want to introduce a breaking change on this key though.

Reply:
That change seems reasonable 👍

Reply:
Ok. Done in 1eaf50ab30e9e2f4ea4a95ac687fc79092383d12

It is cleaner.